src/test/java/com/redhat/cloud/notifications/routers/UserConfigServiceTest.java

```diff
@@ -91,14 +89,14 @@ void testSettings() {
         String bundle = "insights";
         String application = "policies";
 
-        List<SettingsValueJsonForm> jsonForms = given()
+        SettingsValueJsonForm jsonForm = given()
                 .header(identityHeader)
-                .when().get("/user-config/email-preference")
+                .when().get("/user-config/notification-preference?bundleName=insights")
```

Comment:
```suggestion
                .queryParam("bundleName", bundle)
                .when().get("/user-config/notification-preference")
```
Same for the 4 similar calls below.